packages/base/card-api.gts

```diff
@@ -2214,10 +2212,10 @@ export async function getIfReady<T extends CardBase, K extends keyof T>(
       let card = Reflect.getPrototypeOf(instance)!
         .constructor as typeof CardBase;
       let field: Field = getField(card, fieldName as string)!;
-      return field.handleNotLoadedError(instance, e, opts) as
-        | T[K]
-        | T[K][]
-        | undefined;
+      return (await field.handleNotLoadedError(instance, e, {
+        loadFields: true,
+        ...opts,
+      })) as T[K] | T[K][] | undefined;
```

Comment:
Does adding `loadFields: true` inside the host code [here](https://github.com/cardstack/boxel/blob/09c920737f6b01eea02f8b14713c080c09ac966f/packages/host/app/services/card-service.ts#L84) fix the issue?

Reply:
I like this solution much better. Reverted my change and implemented this.